modules/cnf-configuring-amq-messaging-bus.adoc

```diff
@@ -0,0 +1,194 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-configuring-amq-messaging-bus_{context}"]
+= Configuring the AMQ messaging bus for PTP fast event notifications
+
+To pass fast event notifications between publisher and subscriber on a node, you must install and configure an AMQ messaging bus router to run locally on the node.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace custom resource (CR) for the router.
```

Comment:
These is no more valid, sicne we are using amq interconnect installation document

Reply:
Remove this whole section have point to  AMQ interconnect openshift document